### PR TITLE
fix #294495: names of transposed piccolo/clarinet/trumpet

### PR DESCRIPTION
**Pull request by Howard-C**
Fixes https://musescore.org/node/294495.

The standard name should be "piccolo/clarinet/trumpet in xx" instead of "xx piccolo/clarinet/trumpet".

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
see also #5232, which did this for Horns

**Comment by Jojo-Schmitz** [comment]
There an mtest failing, check https://travis-ci.org/musescore/MuseScore/jobs/585143516#L4295-L4313
one of the test files needs to get change to the new name of "Trumpet in B♭"

**Comment by MarcSabatella** [comment]
Would the templates not need to be updated as well?  Although, see my comment in the issue thread.  I don't recall any user requests for this change, and I know the current style (Bb Trumpet, etc) is common as well.  For example, Gould shows "Trumpets in Bb" in her orchestral example, but "Bb Trumpets" in her concert band example.  it's not clear if she is saying that is the relevant distinction or just showing that both are accepted.

Bottom line: I think more discussion on the forum is warranted.

**Comment by Howard-C** [comment]
I modified the failed mtest file and also the windxtet, brassxtet and orchestral templates.
